.changelog/10868.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+service: Removed duplicate name checking for `service` and `task` to prevent sprurious validation failures for services using interpolation or using tags for differentiating versions of a service.
```

Comment:
```suggestion
consul: Fixed a bug where services may incorrectly fail conflicting name validation
```

Trying to make it short so it's still one line in the changelog. Previously we've put service related things under the `consul:` prefix.